package.json

```diff
@@ -104,13 +102,15 @@
   "homepage": "https://github.com/webpack/webpack",
   "main": "lib/index.js",
   "bin": "./bin/webpack.js",
+  "types": "types.d.ts",
```

Comment:
This should be released in a major change. This can break so many things. Before releasing this we should also remove all types from `@types/webpack` and convert it to a "stub types package". That way users can depend on a new major version of webpack and `@types/webpack` (or remove `@types/webpack`) to get these new types without conflicting types.

Reply:
yes it's released as major change.